tests/robustness/failpoints.go

```diff
@@ -338,9 +338,11 @@ func (t triggerDefrag) Available(e2e.EtcdProcessClusterConfig, e2e.EtcdProcess)
 	return true
 }
 
-type triggerCompact struct{}
+type triggerCompact struct {
+	waitForRevision bool
```

Comment:
`waitForRevision` was a working name. Something like `multiBatchCompaction` would be better as we are basically waiting for revision to be big enough so compaction needs to be done in multiple batches.

Reply:
`multiBatchCompaction` looks better and more accurate. let me update it.